.sass-lint.yml

```diff
@@ -77,7 +77,7 @@ rules:
   hex-notation: 2
   indentation: 2
   leading-zero: 2
-  nesting-depth: 2
+  nesting-depth: 3
```

Comment:
This had to be adjusted because of complex css rules to override on hover button color.